lib/config/definitions.ts

```diff
@@ -1284,6 +1284,14 @@ const options: RenovateOptions[] = [
     allowedValues: ['branch', 'pr', 'pr-comment'],
     default: 'pr',
   },
+  {
+    name: 'automergeStrategy',
+    description:
+      'The merge strategy to use when automerging PRs. Used only if automergeType=pr or automergeType=pr-comment',
```

Comment:
```suggestion
      'The merge strategy to use when automerging PRs. Used only if <code>automergeType=pr</code> or <code>automergeType=pr-comment</code>',
```

@viceice I was thinking maybe we could do this? I don't know if this is valid code though for this file?

Reply:
Ah, I'll need to change this anyway — `automergeType=pr-comment` doesn't result in a merge happening (And so the merge strategy doesn't apply). I'll leave this open for the approval or not of the `code` tags, but I'll need to resolve anyway by removing the automergeType=pr-comment bit.

Reply:
```suggestion
      'The merge strategy to use when automerging PRs. Used only if `automergeType=pr`',
```

Should work too. 🙃